Tests/FluentAssertions.Specs/Primitives/DateTimeAssertionSpecs.HaveMillisecond.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using Xunit;
+using Xunit.Sdk;
+
+namespace FluentAssertions.Specs.Primitives;
+
+public partial class DateTimeAssertionSpecs
+{
+    public class HaveMillisecond
+    {
+        [Fact]
+        public void When_asserting_subject_datetime_should_have_milliseconds_with_the_same_value_it_should_succeed()
```

Comment:
🔧 For a while, we've moved away from `When` `Should` constructs in test names and switch to more fact-based naming. See `MemberMatchingSpecs` for example.